boards/toradex/verdin_imx8mm/doc/index.rst

```diff
@@ -0,0 +1,268 @@
+dedicated interfaces while still being able to access the regular interfaces. This heterogeneous
+multicore system allows for the running of additional real-time operating systems on the M4 cores
+for time- and security-critical tasks.
+
+- Board features:
+
+  - RAM: 1GB - 2GB (LPDDR4)
+  - Storage:
+
+    - 4GB - 17GB eMMC
+    - 2KB I²C EEPROM
+  - Wireless:
+
+    - WiFi Dual-band 802.11 ac/a/b/g/n 2.4/5 GHz
+    - Bluetooth 5/BLE
+  - USB:
+
+    - 1x USB2.0 Host
+    - 1x USB2.0 OTG
+  - Ethernet Gigabit
+  - Interfaces:
+
+    - MIPI DSI 1x4 Data Lanes
+    - PCIe Gen 2
+    - MIPI CSI-2
+    - SPI
+    - QSPI
+    - UART
+    - I²C
+    - I²S
+    - SD/SDIO/MMC
+    - GPIOs
+    - CAN
+    - ADC
+    - S/PDIF
+  - Debug
+
+    - JTAG 10-pin connector
+
+More information about the board can be found at the
+`Toradex website`_.
+
+Supported Features
+==================
+
+The Zephyr verdin_imx8mm_m4 board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| CLOCK     | on-chip    | clock_control                       |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | GPIO output                         |
+|           |            | GPIO input                          |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+:zephyr_file:`boards/arm/verdin_imx8mm_m4/verdin_imx8mm_m4_defconfig`.
+
+It is recommended to disable peripherals used by the M4 core on the Linux host.
+
+Other hardware features are not currently supported by the port.
+
+Connections and IOs
+===================
+
+UART:
+
+Zephyr is configured to use the UART4 by default, which is connected to the FTDI
+USB converter on most Toradex carrier boards.
+
+This is also the UART connected to WiFi/BT chip in modules that have the WiFi/BT
+chip. Therefore, if UART4 is used, WiFI/BT will not work properly.
+
+If the WiFi/BT is needed, then another UART should be used for Zephyr (UART2 for
+example). You can change the UART by changing the ``zephyr,console`` and
+``zephyr,shell-uart`` in the :zephyr_file:`boards/arm/verdin_imx8mm_m4.dts` file.
```

Comment:
File path needs updating

Reply:
This has been addressed in the latest push.